Index: cartpole/ppo_change_mec.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cartpole/ppo_change_mec.py b/cartpole/ppo_change_mec.py
--- a/cartpole/ppo_change_mec.py	(date 1668347561067)
+++ b/cartpole/ppo_change_mec.py	(date 1668347561067)
@@ -264,7 +264,7 @@
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('--n_episodes', type=int, default=300)
+    parser.add_argument('--n_episodes', type=int, default=500)
     parser.add_argument('--len_episode', type=int, default=512)
     parser.add_argument('--lr', type=float, default=0.0001)
     parser.add_argument('--batch', type=int, default=64)
